src/aks-preview/azext_aks_preview/managed_cluster_decorator.py

```diff
@@ -317,6 +317,20 @@ def get_sku_name(self) -> str:
                 skuName = CONST_MANAGED_CLUSTER_SKU_NAME_BASE
         return skuName
 
+    def _get_enable_addons(self, enable_validation: bool = False) -> List[str]:
+        enable_addons = super()._get_enable_addons()
+        sku_name = self.raw_param.get("sku")
```

Comment:
I think `self.get_sku_name()` would be better than `self.raw_param.get("sku")`

Reply:
Updated to use self.get_sku_name()